src/R/Editor/Impl/Data/WorkspaceVariableProvider.cs

```diff
@@ -85,7 +85,7 @@ internal sealed class WorkspaceVariableProvider : RSessionChangeWatcher, IVariab
                 }
 
                 // May be a package object line mtcars$
-                variableName = TrimToTrailingSelector(variableName);
+                variableName = TrimToFirstSelector(variableName);
```

Comment:
Actually, come to think of it, doesn't it do the wrong thing now? If I understand correctly, something like `a$b$` will now be trimmed to the first `$`, so `variableName` becomes `a`. And then you call `DescribeChildrenAsync` on that below. So completion list would that for members of `a`, not for members of `a$b`?

It sounds like it should instead not check `exists()` at all, and instead feed the entire expression up until trailing selector to `DescribeChildrenAsync`, and check if the latter returns an error. If it does, then make member list is empty.